vms/platformvm/warp/test_signer.go

```diff
@@ -23,7 +23,7 @@ func TestSignerWrongChainID(t *testing.T, s Signer, _ *bls.SecretKey, _ ids.ID)
 	require := require.New(t)
```

Comment:
I feel like there should now be a `TestSignerWrongNetworkID` test (which will currently fail). Technically we're only adding the `networkID` to de-duplicate the P-chain... so I guess it isn't really required... but it feels like it should be here.

Reply:
Updated to add a unit test for the signer and signature verification